packages/lesswrong/components/moderationTemplates/ModerationTemplatesPage.tsx

```diff
@@ -34,27 +49,74 @@ export const ModerationTemplatesPage = ({classes}: {
   
   if (!userCanDo(currentUser, 'moderationTemplates.edit.all')) return null
   
-  const nonDeletedTemplates = moderationTemplates.filter(template => !template.deleted)
-  const deletedTemplates = moderationTemplates.filter(template => template.deleted)
+  const filteredTemplates = moderationTemplates.filter(template => {
+    return template.collectionName === filter || !filter
+  })
+  const nonDeletedTemplates = filteredTemplates.filter(template => !template.deleted)
+  const deletedTemplates = filteredTemplates.filter(template => template.deleted)
+
+  const handleFilter = (type:string) => {
```

Comment:
this probably also wants to be typed `type: TemplateType` rather than `type: string`